packages/plugin-engages-ui/src/ActivityLog.tsx

```
@@ -0,0 +1,105 @@
+        )}
+        {bcc.length > 0 && (
+          <>
+            {", "}
+            {__("bcc:")}
+            <strong>{bcc.join(", ")}</strong>
+          </>
+        )}
+      </SentWho>
+
+      <IMapActivityContent $shrink={shrink}>
+        <div dangerouslySetInnerHTML={{ __html: body }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)